traitsui_user_manual/factories_basic.html

```diff
@@ -95,8 +117,7 @@
 </tr>
 <tr class="field-even field"><th class="field-name">Default for:</th><td class="field-body">Array, CArray (if 2-D)</td>
 </tr>
-<tr class="field-odd field"><th class="field-name" colspan="2">Optional parameter:</th></tr>
-<tr class="field-odd field"><td>&nbsp;</td><td class="field-body"><em>width</em></td>
+<tr class="field-odd field"><th class="field-name">Optional parameter:</th><td class="field-body"><em>width</em></td>
```

Comment:
I changed my local `distutil.html4css1` "field-name-limit" to 20 in order to remove the `colspan="2"` here.  It is a hack and I decided to do so because otherwise the page would look like this under the new theme:
![screen shot 2016-04-15 at 15 53 34](https://cloud.githubusercontent.com/assets/3673984/14565318/714077ba-0322-11e6-9d6b-cfc73728de77.png)
Now it would look like this
![screen shot 2016-04-15 at 15 55 49](https://cloud.githubusercontent.com/assets/3673984/14565335/91fa82ca-0322-11e6-9d84-111eab477a6b.png)
The old field-name-limit (and hence colspan=2) had made sense with the old theme but not with the new one.  Not sure if we want to modify the theme or add setting for `field-name-limit`.  I did some preliminary research for the latter and it does not seem trivial.